### PR TITLE
Adding support for query type contains

### PR DESCRIPTION
**Pull request by gesundkrank**
Added query type `contains` and filter type `search`.

Unfortunately had to drop support of parsing query arguments to array. Since I introduced the `caseSensitive` parameter. But I would also be happy if you come up with a cooler solution. 

Thanks!!

### CODE REVIEW

**Comment by estliberitas** [comment]
Okay, will check & release in 1-2 days.

**Comment by estliberitas** [comment]
LGTM

File: lib/fields/query.js [hunk 1]
```diff
@@ -34,12 +35,20 @@ function query(type, value) {
   // FragmentSearchQuerySpec
   else if (type === 'fragment') {
     if (!Array.isArray(value)) {
-      value = utils.args(arguments, 1)
+      throw new FieldError('value is no array')
```
Comment (estliberitas) [inline lib/fields/query.js]: s/no/not/

**Comment by estliberitas** [comment]
Can you split these two commits into two different PRs? Coz it does not look like they are logically related. At least, to me.

File: lib/fields/filters/search.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+var FieldError = require('../../errors').FieldError
+
+
+/**
+ * Search filter
+ *
+ * @see http://druid.io/docs/0.9.0/querying/filters.html
+ *
+ * @param {string} dimension Dimension to which filter is applied
+ * @param {object} query Query to filter by.
```
Comment (estliberitas) [inline lib/fields/filters/search.js]: s/.//

**Comment by estliberitas** [comment]
> Can you split these two commits into two different PRs? Coz it does not look like they are logically related. At least, to me.

I.e. you can leave this PR for `contains` query type (since it's the first commit), and create another one for `search` filter type

**Comment by gesundkrank** [comment]
Ok. Will open the new pull request after this has been merged.

**Comment by estliberitas** [comment]
@gesundkrank then do it, man 😉

**Comment by gesundkrank** [comment]
Done. #19

**Comment by estliberitas** [comment]
@gesundkrank 2.0.0 released

**Pull request merged**